spikingjelly/activation_based/triton_kernel/spikezip_kernel.py

```
@@ -0,0 +1,226 @@
+
+from .triton_utils import (
+    register_op,
+    type_dict,
+    use_static_range_for_triton_neuron_kernel,
+    wrap_triton,
+)
+
+try:
+    import triton
+    import triton.language as tl
+except BaseException as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `BaseException`

(BLE001)